app/server/models/organisation.py

```diff
@@ -55,3 +59,9 @@ def __init__(self, **kwargs):
             wei_target_balance=current_app.config['SYSTEM_WALLET_TARGET_BALANCE'],
             wei_topup_threshold=current_app.config['SYSTEM_WALLET_TOPUP_THRESHOLD'],
         )
+
+    def send_welcome_sms(self, to_user):
+        if self.custom_welcome_message_key:
+            message = i18n_for(to_user, "organisation.{}".format(self.custom_welcome_message_key))
+            send_message(to_user.phone, message)
```

Comment:
any idea how i can mock out the `send_message` method here for a test? given it's not a method on the class itself but from elsewhere. if you look at the test below i did for kenya state machine, i was hoping to do the same for a test for this, but i feel like there's some pythonfoo that i'm lacking here

Reply:
Perhaps we'd have a sms sender class that we inject as a dependency on app buildup, which we can swap to a mock for tests?

Reply:
@wangfowen Something like this?
https://github.com/teamsempo/SempoBlockchain/blob/44e21e44ece12e6ec8c797eef7eeb0ead9e8fb14/test/unit/conftest.py#L92
(I believe requires wrapping the sms utils in a class, as monkeypatching raw functions inside a module didn't work for me) 

Alternatively something like this?
https://github.com/teamsempo/SempoBlockchain/blob/88a01d45cf9d275bc002b6e3ff89051e8e8d6223/test/functional/app/me_api/test_me_misc.py#L61